src/get-subseqs.py

```diff
@@ -18,8 +72,17 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
-    print(f"and the coordinates in {args.coords}")
+    # print(f"Now I need to process the records in {args.fasta}")
+    # Read the content
+    content = args.fasta.read()
+    genomes = getGenomes(content)
```

Comment:
Put the genomes in a `dict` here for easier (and faster) lookup:

```python
genomes = {name: seq for name, seq in getGenomes(content)}
```

or something like that.